### PR TITLE
fix systemjs example

### PR DESCRIPTION
**Pull request by SamVerschueren**
Upgraded the SystemJS example. Tried to upgrade the Webpack example as well but received an error, no idea why though but I'm not a Webpack user so hard to troubleshoot :).

### CODE REVIEW

File: examples/systemjs/src/app.component.ts [hunk 1]
```diff
@@ -15,12 +15,14 @@ import {TranslateService} from 'ng2-translate/ng2-translate';
...
   `,
 })
-export class AppComponent {
-    constructor(private translate: TranslateService) {
-        translate.addLangs(["en", "fr"]);
-        translate.setDefaultLang('en');
+export class AppComponent implements OnInit {
+    constructor(private translate: TranslateService) { }
+  
+    ngOnInit() {
```
Comment (SamVerschueren) [inline examples/systemjs/src/app.component.ts]: Used `ngOnInit` because it's best practice to keep the constructor as light as possible.

**Comment by ocombe** [comment]
Could you update it to 4.0.0? the new main file is `bundles/ng2-translate.umd.js` (the index.js still exists for compatibility)

**Comment by SamVerschueren** [comment]
Done!

**Comment by ocombe** [comment]
thanks !

**Comment by SamVerschueren** [comment]
Cool! Excited that all these things are being improved step by step :).

**Pull request merged**